src/e2e/features/2-zaken-toevoegen.feature

```diff
@@ -4,5 +4,7 @@ Feature: Zaak
     Given "Bob" navigates to "zac" with path "/"
     When "Bob" wants to create a new zaak
     Then "Bob" sees the created zaak with a delay
+    When "Bob" clicks on Create Document for zaak
```

Comment:
what are we testing here? reading this scenario it seems like we are testing opening the create document and closing the create document tab?

Reply:
Also it may be nice to move this to a seperate scenario, so the tests are not dependent from eachother
You can access the newly created zaak in test-storage with `this.testStorage.get('caseNumber')`

Reply:
We're testing if we can open SmartDocuments. 
Accessing the zaak from testStorage works well, but we wanted to detect if SmartDocs integration is working. We had an issue with certificates and e2e did not cover this, hence the changes I made.

We can extend it later to cover the real docs (create/attach/...)

@RickWoltheus WDYT? Should we still go for a separate scenario?

Reply:
Added a new `external-systems.feature`